int32 phi
int32 point_num
int32 record_item
float64 Rx
float64 Ry
float64 Rz